### PR TITLE
chore: capture JSON-RPC error

### PR DESCRIPTION
**Pull request by wa0x6e**
## 🧿 Current issues / What's wrong ?

`capture` is expecting an error object, whereas the snapshot.js SDK is returning a JSON-RPC  object on some occasion, resulting on this kind of generic error: `Non-Error exception captured with keys: code, data, message` (https://snapshot-labs.sentry.io/issues/4452136608/?project=4505606761152512&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=14d&stream_index=8)

## 💊 Fixes / Solution

Convert the JSON-RPC object to an error before sending it to sentry, with a better descriptive error message, and contexts.

## 🚧 Changes

- Convert and format the JSON-RPC object as an error, before sending to Sentry

## 🛠️ Tests

- N/A

### CODE REVIEW

File: src/writer/proposal.ts [hunk 1]
```diff
@@ -141,8 +141,20 @@ export async function verify(body): Promise<any> {
...
+      } else {
+        capture(new Error('Validation failed'), {
+          contexts: {
+            validationError: e,
+            input: {
+              space: msg.space,
+              address: body.address
+            }
+          }
+        });
```
Comment (ChaituVR) [inline src/writer/proposal.ts:156]: Could we handle this in `capture` itself? maybe a new params `options` with something like `defaultError` inside it,

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/writer/proposal.ts (hunk 1 above)
Comment (wa0x6e) [reply src/writer/proposal.ts:156]: Sounds like a good idea 👍

**Comment by wa0x6e** [comment]
Will be handled by snapshot-sentry package

**Comment by wa0x6e** [comment]
See https://github.com/snapshot-labs/snapshot-sentry/pull/8

**Pull request closed without merging**